inbound_parse.go

```diff
@@ -73,7 +73,7 @@ func sendGridHandler(c *gin.Context) {
 
 	wiiMail, err := FormulateMail(fromAddress.Address, toAddress, c.PostForm("subject"), text, attachedFile)
 	if err != nil {
-		log.Printf("error formulating mail: %v", err)
+		log.Printf("Error formulating mail: %v", err)
```

Comment:
errors in Go should be lower case. This may not be `errors.New` but we're treating it as such.
It's not fatal however, so we use `log.Printf`. Perhaps it should be wrapped to Sentry instead?